### PR TITLE
CP / CP-2 for Redis and Elasticsearch

### PR DESCRIPTION
**Pull request by pburkholder**
## Changes proposed in this pull request:

- CP-2 for AWS redis/es brokering

:sunglasses:[PREVIEW URL](https://cg-88d42ca6-59d7-47e0-9500-4dd9251360b9.app.cloud.gov/preview/cloud-gov/cg-site/BRANCH_NAME)

## Security Considerations

The services we broker into AWS govcloud are public and non-sensitive.

### CODE REVIEW

File: _docs/ops/contingency-plan.md [hunk 1]
```diff
@@ -130,6 +130,17 @@ If all AWS regions are disrupted, Cloud Operations will deploy the system to ano
 #### AWS RDS
 Major services (Cloud Foundry, BOSH, Concourse, and UAA) rely on databases provided by RDS. To restore a database from backup, see [restoring RDS]({{ site.baseurl }}{% link _docs/ops/runbook/restoring-rds.md %}).
 
+#### Brokered AWS services
+
+cloud.gov is not dependent on the following services, but we do broker customer provisioning and access to these services, so our customers could be impacted if they were unavailable.  
+
+If a GovCloud service were wholly unavailable, we would broker access to the AWS commercial cloud equivalent, as shown below:
```
Comment (ccostino) [inline _docs/ops/contingency-plan.md]: Are we truly able to do this?

**Review by ccostino — COMMENTED** [review]
Thanks, @pburkholder!  The changes look fine, but I had one question regarding the viability of doing this; mainly, are we truly able to broker services in the commercial side?  I didn't think we could.

File: _docs/ops/contingency-plan.md (hunk 1 above)
Comment (pburkholder) [reply _docs/ops/contingency-plan.md]: There would be engineering work regarding the authorization of the broker (across account boundaries) and the services would no longer be in the same logical VPC, so we'd have to make the call whether our customers can accept those risks, and communicate those clearly.

Comment (ccostino) [reply _docs/ops/contingency-plan.md]: Okay, that sounds reasonable; do we also need to make a note about receiving approval from our AO for this change to occur such as the point above about redeploying in another IaaS environment, or is that out of scope for this?

**Review by ccostino — APPROVED** [review]
Looks good to me, thanks, @pburkholder!

**Pull request merged**